src/types.ts

```diff
@@ -52,6 +52,7 @@ export interface BidsDto {
 export interface Bid extends BidParams {
   date: number;
   signedMessage: string;
+  version?: string
```

Comment:
Consider making a new interface `MongoBid` which has version as nullable. And then making `Bid` have version but it is not nullable.

Then change all of the database adapter code so that it types the returned value as `MongoBid`.

Then create a conversion function `MongoBidToBid` (or something) that will fill `version` in with `"1.0"` if no version is found.

This way the API has a smoother surface. Version is always defined and we've hidden away the fact that the database may or may not have the version stored in it. This makes SDK integration easier since now you don't have to check if version is nullable.

(We are going to check if it's nullable someplace, so let's do it as high up stream as we can)